src/App.jsx

```diff
@@ -14,70 +16,83 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const noGoodsSelected = 'No goods selected';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+const checkSelected = (good, goodValue) => goodValue === good;
```

Comment:
There is no need to create a function for that. You can just save this comparison as a variable on mapping
`const isSelected = good === selectedGood`